frontend/src/app/main/render.cljs

```diff
@@ -41,7 +370,7 @@
 (defn populate-fonts-cache [objects]
   (let [texts (->> objects
                    (vals)
-                   (filterv text?)
+                   (filterv #(= (:type %) :text))
```

Comment:
Is there any reason for this change?

Reply:
yes and no, text? predicate is not used anywhere, so for so small function I considered remove it from toplevel.